qutebrowser/app.py

```diff
@@ -660,9 +660,9 @@ def restart(self, pages=(), session=None, override_args=None):
         try:
             args, cwd = self._get_restart_args(pages, session, override_args)
             if cwd is None:
-                subprocess.Popen(args)
+                subprocess.run(args)
             else:
-                subprocess.Popen(args, cwd=cwd)
+                subprocess.run(args, cwd=cwd)
```

Comment:
Turns out `subprocess.Popen` was actually the right thing to use here, as we *don't* want to wait for the child process at all here. I fixed this in 4d8ac7486ccdd512b9a4f5d753be79aef418478c.